test_scripts/API/Restructuring_OnResetTimeout/017_BC_OnResetTimeout_with_invalid_data.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 3000) with right requestID but wrong methodName to SDL
+-- 4) HMI does not send response
+-- SDL does:
+-- 1) Respond in 10 seconds with GENERIC_ERROR resultCode to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local pOnResetTO = {
+	missingMandatory = { resetPeriod = 3000 },
+	outOfBounds = { requestID = 65536, methodName = 111, resetPeriod = 1000001 },
```

Comment:
in this struct you have invalid requestID, invalid methodName and also invalid resetPeriod, one step must check only one wrong parameter. the same comment to parameters below

Reply:
@GetmanetsIrina Done